src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -814,7 +814,13 @@
                                   Header="{x:Static p:Resources.DynamoViewDebugMenuDumpLibrary}"
                                   Command="{Binding DumpLibraryToXmlCommand}"
                                   IsEnabled="True" />
-                    </MenuItem>
+                        <MenuItem Focusable="False"
+                                  Name="DebugModes"
+                                  Header="{x:Static p:Resources.DynamoViewDebugMenuDebugModes}"
+                                  Click="OnDebugModesClick"
+                                  DataContext="{Binding Path=Model.UpdateManager}"
```

Comment:
This does not look like the correct DataContext?

Reply:
nice catch